.github/workflows/rc-release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release to @rc-<name> tag on npm
+
+on:
+  push:
+    branches:
+      - "rc/*"
+
+jobs:
+  release-pr:
+    name: "Release RC to npm"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Ensure NPM access
+        run: npm whoami
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Build
+        run: pnpm build
+
+      - name: Get RC name
+        uses: frabert/replace-string-action@v2
+        id: rc_name
+        with:
+          string: ${{ github.ref }}
+          pattern: "rc/"
+          replace-with: ""
```

Comment:
```suggestion
          string: ${{ github.ref }}
          pattern: "rc/"
          replace-with: "rc-"
```

Reply:
This will require removing all duplicated usages of the `rc-` prefix.

Reply:
Is this regarding the unpublish script?

Reply:
Or do you mean, when a new version is published, previous snapshots are deleted?

Reply:
I was referring only to the `rc-` prefix string used in the rest of the _code_.

I will make a new/complete suggestion to illustrate what I mean. :)

Reply:
Ah, I understand

Reply:
Resolved [https://github.com/FuelLabs/fuels-ts/pull/1497/commits/608ad9b8e20cef1699d1dc55f6880fb6f4644f0c](url)